src/main/java/uk/gov/hmcts/reform/bsp/common/model/update/in/CaseDetails.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.bsp.common.model.update.in;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.Map;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder(toBuilder = true)
+public class CaseDetails {
+
+    @JsonProperty("case_data")
+    private Map<String, Object> caseData;
+
+    /**
+     * This field is expected in the future, please see: https://tools.hmcts.net/jira/browse/BPS-968
+     * but we can add it now as we `JsonIgnoreProperties(ignoreUnknown = true)`
+     * */
+    @JsonProperty("id")
```

Comment:
This change isn't official in Bulk Scanning, yet.

Reply:
ok, the field is not mandatory